packages/datawell/css.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/css/index.js')
+module.exports = require('./dist/cjs/css/index.js')
```

Comment:
I'm glad we have BREAKING CHANGE marked in the commit message.  But it's not very user friendly.  This will go in the changelog.md, and in order to be useful must indicate enough to the user to understand.  

The message you created for button is much better: https://github.com/pluralsight/design-system/blob/master/packages/button/CHANGELOG.md#breaking-changes

Something like that would be nice because it's more specific and understandable.  This is more mysterious.  Though it will create the correct major version bump.

Maybe:

> a named package import now imports the React component

What do you think?

Reply:
For current usage, documented patterns, this doesn't really break anything for anyone honestly. Because the patterns we use always point to `(react|css|js|vars).js`.  Those cjs files still exist as they always have. I merely updated the path to on the root re-exports so that this upgrade was backward compatible for people using the CJS import pattern we've been following until now.

All of the ESM mods will only effect someone who is explicitly importing the package using the main module which until now was always an object with one or more of the following keys:
- react
- css
- vars
- js


However I have been working on a release post that will explain how to migrate to the new import pattern to take advantage of treeshaking. The important thing is to make that clear in the release notes so I'll probably run them by Dane on Monday to get his feedback.